htdocs/bootstrap/css/custom-css.css

```diff
@@ -340,6 +340,7 @@ div.navbar div.container div.navbar-brand {
     color: #064785;
     background-color: #E4EBF2;
     border-color: #E4EBF2;
+    margin: 0px 14px;
```

Comment:
Inside CCS-land you don't need "px" or whatever measurment when the value is 0.
You can use this instead `margin: 0 14px;` and it just makes it a little cleaner. Unless you really write your own CSS a lot, you won't be bothered by it.